shell/server/proxy.js

```diff
@@ -1604,7 +1598,7 @@ Proxy.prototype.handleRequest = function (request, data, response, retryCount) {
     var path = request.url.slice(1);  // remove leading '/'
     var session = self.getSession(request);
 
-    if (request.method === "GET") {
+    if (request.method === "GET" || request.method === "HEAD") {
```

Comment:
The HTTP spec says that HEAD is supposed to be exactly the same as a GET, but the body shouldn't be sent back. I thought I'd save having to add HEAD to web-session, but if it's clearer to do so then that'd be fine as well.